src/main/webapp/scripts/appscript.js

```diff
@@ -106,6 +116,12 @@ function validateEditFormInput() {
   }
 }
 
+function addAccount() {
+  if(buildEditForm()) {
+    validateEditFormInput();
+  }
```

Comment:
It doesn't look like buildEditForm() returns a boolean here. What does this do?

Reply:
I forgot that I wasn't using this method anymore. I'll delete that.

Reply:
I'm not using it anymore, and I forgot to delete it. I'll go do that now.